juha/programmers/5/이상한-문자-만들기.js

```diff
@@ -0,0 +1,17 @@
+function solution(s) {
+  let answer = "";
+  const upperStr = s.toUpperCase();
+  const lowerStr = s.toLowerCase();
+  let cnt = 1;
```

Comment:
변수명, 함수명 등은 직관적으로 알기 쉽게 작명하는 것이 좋습니다.
```suggestion
  let count = 1;
```